src/client/pydaos/raw/daos_api.py

```diff
@@ -2282,7 +2287,7 @@ def __init__(self, path):
             'close-obj':       self.libdaos.daos_obj_close,
             'close-tx':        self.libdaos.daos_tx_close,
             'commit-tx':       self.libdaos.daos_tx_commit,
-            'connect-pool':    self.libdaos.daos_pool_connect,
+            'connect-pool':    self.libdaos.daos_pool_connect2,
```

Comment:
daos_pool_connect?

Reply:
Right. Johann suggested to exposed daos_pool_connect2() only. Other languages need to call daos_pool_connect2.